app/lib/BaseModelWithAttributes.php

```diff
@@ -2199,7 +2199,10 @@ public function htmlFormElementForAttributeSearch($po_request, $pm_element_code_
 			$pa_options['format'] = $vs_format;
 			
 			if ($vs_rel_types = join(";", caGetOption('restrictToRelationshipTypes', $pa_options, array()))) { $vs_rel_types = "/{$vs_rel_types}"; }
-			
+
```

Comment:
line 2282 and 2290 is where the issue is actually caused. Alternate solution could be to use $vs_fld_name instead of $f, but seems more likely to cause additional issues?